app/routes.js

```diff
@@ -3,192 +3,328 @@
+
+            var notifyClient = new NotifyClient("conor_preprod_api_key-6c19e868-f026-4ff4-86ed-8effb112c0cc-f0323ce0-45bc-4cbb-8976-dc58b24a0f75")
+
+            if (req.body.user_ref !== "undefined" && req.body.user_ref !== null && req.body.user_ref !== "") {
+                if (req.body.service_type == 1) {//standard service
+
+                    notifyClient
+                        .sendEmail("1f0557da-4c66-4f05-95ff-1ea55daede9f", req.body.to, {
+                            personalisation: {
+                                'application_reference': req.body.application_reference,
+                                'email_address': req.body.to,
+                                'user_ref':req.body.user_ref
+                            },
+                            reference: "application submission standard"
+                        })
+                        .then(response => console.log(response))
+                        .catch(err => console.error(err))
+
+
+                    console.log('Sending submission confirmation email');
+                } else {
+                    (req.body.service_type == 2)
+                    {//premium and drop-off service
+
+                        notifyClient
+                            .sendEmail("2fa30eab-e847-4fc0-ae2b-1725923adb9a", "c.mcgandy@kainos.com", {
+                                personalisation: {
+                                    'application_reference': req.body.application_reference,
+                                    'user_ref':req.body.user_ref
+
+                                },
+                                reference: "app submission notify test"
+                            })
+                            .then(response => console.log(response))
+                            .catch(err => console.error(err))
+
+
+                        console.log('Sending submission confirmation');
+                    }
+                }
+            }else{
+                if (req.body.service_type == 1) {//standard service
+
+                    notifyClient
+                        .sendEmail("03acf3ba-0c95-438a-9ead-f6daadb8bb93", req.body.to, {
+                            personalisation: {
+                                'application_reference': req.body.application_reference,
+                                'email_address': req.body.to
+                            },
+                            reference: "application submission standard"
+                        })
+                        .then(response => console.log(response))
+                        .catch(err => console.error(err))
+
+
+                    console.log('Sending submission confirmation');
+                } else {
+                    (req.body.service_type == 2)
+                    {//premium and drop-off service
+
+                        notifyClient
+                            .sendEmail("6bc36b7a-dbd9-4363-b188-b3eed8c4fc79", "c.mcgandy@kainos.com", {
```

Comment:
Move this out into the config .env file, replace hard coded email address with request email address